etc/rc.d/nfsd

```diff
@@ -13,13 +13,15 @@ name="nfsd"
 desc="Remote NFS server"
 rcvar="nfs_server_enable"
 command="/usr/sbin/${name}"
+nfs_server_vhost=""
 
 load_rc_config $name
 start_precmd="nfsd_precmd"
 sig_stop="USR1"
 
 nfsd_precmd()
 {
+	local	_vhost
```

Comment:
I am confused where/why is this var used?

Other than that looks OK to me

Reply:
D'oh.  Hold-over from one of the interim versions of the startup script.  Removed, thanks!